client/state/initial-state.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import debugModule from 'debug';
+import localforage from 'localforage';
+
+/**
+ * Internal dependencies
+ */
+import { createReduxStore } from 'state';
+import { reducer } from 'state';
+import { TO_OBJECT, FROM_OBJECT } from 'state/action-types'
+
+/**
+ * Module variables
+ */
+const debug = debugModule( 'calypso:state' );
+
+const localforageConfig = {
+	name: 'calypso',
+	storeName: 'calypso_store',
+	description: 'Calypso Storage'
+};
+
+function toObject( state ) {
+	return reducer( state, { type: TO_OBJECT } );
```

Comment:
Is a noun prefix necessary for actions that affect the whole tree? For example is it `STATE_SERIALIZE` or simply `SERIALIZE`?